### PR TITLE
Add all WP default file types to uploader

### LINKED ISSUE #112: `import files` should handle more file types by default
It would be nice if most standard audio, video, and document extensions were supported. Maybe even all files supported by `get_allowed_mime_types()`?

### LINKED ISSUE #113: `import files` should flag extensions that were skipped/ignored
Maybe after all files have been scanned, the import can show a summary of the import including counts of extensions including ones that will be skipped because they are not included in the `types` list.

### PR DESCRIPTION
**Pull request by joshbetz**
Fixes #112

### CODE REVIEW

**Comment by joshbetz** [comment]
We may want to reconsider flv and maybe js.

File: src/bin/vip-import.js [hunk 1]
```diff
@@ -21,10 +21,77 @@ function list(v) {
...
+	'mdb','mpp',
+	'docx','docm','dotx','dotm',
+	'xlsx','xlsm','xlsb','xltx','xltm','xlam',
+	'pptx','pptm','ppsx','ppsm','potx','potm','ppam',
+	'sldx','sldm',
+	'onetoc','onetoc2','onetmp','onepkg','oxps',
+	'xps',
+	'odt','odp','ods','odg','odc','odb','odf',
+	'wp','wpd',
+	'key','numbers','pages'
```
Comment (mjangda) [inline src/bin/vip-import.js]: Trailing comma?

Comment (mjangda) [inline src/bin/vip-import.js]: Trailing comma?

Comment (mjangda) [inline src/bin/vip-import.js]: Trailing comma?

**Comment by mjangda** [comment]
> We may want to reconsider flv and maybe js.

If we do this, we should probably block it in mu-plugins as well. I'm not sure if we really need to though?

**Pull request merged**